keras_nlp/models/xlm_roberta/xlm_roberta_masked_lm_preprocessor_test.py

```diff
@@ -0,0 +1,175 @@
+# Copyright 2023 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for  XLMRoBERTa masked language model preprocessor layer."""
+import io
+import os
+
+import pytest
+import sentencepiece
+import tensorflow as tf
+from absl.testing import parameterized
+from tensorflow import keras
+
+from keras_nlp.models.xlm_roberta.xlm_roberta_masked_lm_preprocessor import (
+    XLMRoBERTaMaskedLMPreprocessor,
+)
+from keras_nlp.models.xlm_roberta.xlm_roberta_tokenizer import (
+    XLMRobertaTokenizer,
+)
+
+
+class XLMRoBERTaMaskedLMPreprocessorTest(
+    tf.test.TestCase, parameterized.TestCase
+):
+    def setUp(self):
+        vocab_data = tf.data.Dataset.from_tensor_slices(
+            ["the quick brown fox", "the earth is round"]
+        )
+
+        bytes_io = io.BytesIO()
+        sentencepiece.SentencePieceTrainer.train(
```

Comment:
for both tests, there is no need to path all of this to the trainer 
just train it like the Custom vocabulary example in XLMRoberta docstring. and XLMRoberta tokenizer will handle all the tokens for you.